test/extended/builds/run_policy.go

```diff
@@ -0,0 +1,241 @@
+package builds
+
+import (
+	"fmt"
+	"strings"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	buildapi "github.com/openshift/origin/pkg/build/api"
+	buildclient "github.com/openshift/origin/pkg/build/client"
+	buildutil "github.com/openshift/origin/pkg/build/util"
+	exutil "github.com/openshift/origin/test/extended/util"
+	kapi "k8s.io/kubernetes/pkg/api"
+)
+
+var _ = g.Describe("[builds][Slow] using build configuration runPolicy", func() {
```

Comment:
@smarterclayton are you OK with doing the conformance tests as follow up? (there is already pretty good test coverage and we have follow up card for this work already in trello).